internal/test/issues/issue-1039/defaultbehaviour/defaultbehaviour_test.go

```diff
@@ -0,0 +1,53 @@
+package defaultbehaviour
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func ptr[T any](v T) *T {
+	return &v
+}
+
+func TestNullableDisabled(t *testing.T) {
```

Comment:
did we want to have the same test shown for the Nullable types, so it's clear the difference in functionality?